flink-staging/flink-scala-shell/src/main/scala/org.apache.flink/api/scala/FlinkShell.scala

```diff
@@ -44,6 +46,16 @@ object FlinkShell {
           c.copy (host = x)
       }  text("host specifies host name of running JobManager")
 
+      opt[(String)] ('a',"addclasspath") action {
+        case (x,c) =>
+          val xArray = x.split(":")
+          var extJarArray = new Array[String](xArray.length)
+          for(i <- 0 to xArray.length - 1){
+            extJarArray(i) = xArray(i)
+          }
```

Comment:
good point, it's a remainder of an older version with a slightly different approach.